### PR TITLE
Adding support of Bootstrap 5

### PR DESCRIPTION
**Pull request by mustafasalahuldin**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by peterblazejewicz** [comment]
@mustafasalahuldin what's the focus of this PR? The title says 'Bootstrap 5'-related update? To which DT definition? What are the other changes for?

File: types/bootstrap-multiselect/index.d.ts [hunk 1]
```diff
@@ -8,12 +8,12 @@
 
 interface Templates {
     button?: string | undefined;
-    ul?: string | undefined;
+    popupContainer?: string | undefined;
```
Comment (sandersn) [inline types/bootstrap-multiselect/index.d.ts:11]: is this related to the jquery-highlight types? If not you should move it into a separate PR.

**Review by sandersn — CHANGES_REQUESTED** [review]

File: types/bootstrap-multiselect/index.d.ts (hunk 1 above)
Comment (mustafasalahuldin) [reply types/bootstrap-multiselect/index.d.ts:11]: yes, but I don't know how to make separate PR for it. I already sent from GitHub desktop two different PR, but for some reason they come as accumulative.

File: types/jquery-highlight/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Installation
```
[the remark this answers is not in the document]
Comment (mustafasalahuldin) [reply types/jquery-highlight/README.md:1]: OK, I'll do.

File: types/bootstrap-multiselect/tslint.json [hunk 3]
```diff
@@ -1 +1 @@
-{ "extends": "@definitelytyped/dtslint/dt.json" }
+{ "extends": "@definitelytyped/dtslint/dt.json"}
```
Comment (sandersn) [inline types/bootstrap-multiselect/tslint.json:1]: please revert this space removal

File: types/bootstrap-multiselect/index.d.ts [hunk 4]
```diff
@@ -3,17 +3,17 @@
 // Definitions by: Mustafa Salaheldin <https://github.com/mustafasalahuldin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
-
+// Adding support to bootstrap 4
```
Comment (sandersn) [inline types/bootstrap-multiselect/index.d.ts:6]: is it bootstrap 4 or 5? Either way this comment seems unneeded.

**Review by sandersn — CHANGES_REQUESTED** [review]
Couple more minor requests

**Comment by mustafasalahuldin** [comment]
not quite sure what is needed here?

**Comment by mustafasalahuldin** [comment]
Please clarify what is required here.

**Comment by sandersn** [comment]
1. Split the jquery-highlight changes into a separate PR.
2. Reconcile the "Adding support to bootstrap 4" comment with the "Adding support of Bootstrap 5" title of this PR. Is it adding support for 4 or 5? Is it necessary to comment which version in the first place?

**Review by gabritto — CHANGES_REQUESTED** [review]
Please address the requested changes in @sandersn's latest comment. Thank you.

File: types/bootstrap-filestyle/index.d.ts [hunk 5]
```diff
@@ -1,30 +1,34 @@
-// Type definitions for bootstrap-filestyle 1.2
-// Project: https://github.com/markusslima/bootstrap-filestyle
+// Type definitions for bootstrap-filestyle 2.1.4
```
Comment (peterblazejewicz) [inline types/bootstrap-filestyle/index.d.ts:1]:
```suggestion
// Type definitions for bootstrap-filestyle 2.1
```
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

File: types/bootstrap-filestyle/index.d.ts [hunk 6]
```diff
@@ -1,30 +1,34 @@
...
+// Project: https://github.com/jcputney/bootstrap-filestyle
 // Definitions by: Mustafa Salaheldin <https://github.com/mustafasalahuldin>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Definitions: https://github.com/jcputney/bootstrap-filestyle
```
Comment (peterblazejewicz) [inline types/bootstrap-filestyle/index.d.ts:4]:
```suggestion
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```
this one looks unnecessary

File: types/bootstrap-filestyle/index.d.ts (hunk 5 above)
Comment (peterblazejewicz) [reply types/bootstrap-filestyle/index.d.ts:1]: as per note, make this agnostic to BS version,
package is old, and seems to be done against BS4:
https://github.com/markusslima/bootstrap-filestyle/blob/071ae6f3ff9a49a04346848b05d165e86c50f3b7/package.json#L4
